### DISCUSSION TITLE
Deleting the last styles in the compiled css, after adding my styles

### DISCUSSION THREAD

**Discussion by maria1186** [tailwindlabs/tailwindcss#8859] [Help]
**What version of Tailwind CSS are you using?**

For example: v3.1.6

**What build tool (or framework if it abstracts the build tool) are you using?**

Pug cli

**What version of Node.js are you using?**

For example: v14.18.0

**What browser are you using?**

 Chrome

**What operating system are you using?**

 Windows

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways. Unfortunately we can't provide support without a reproduction, and your issue will be closed with no comment if this is not provided.

**Describe your issue**

Folder structure: 
  pug - This is pug template
  src - style.css ( @tailwind base;@tailwind components;@tailwind utilities;)
dist - compiled HTML file
After I add my styles after inserting Tailwind  @tailwind base;@tailwind components;@tailwind utilities;)
 I see how in src/style.css  it added my style but deleted last lines from Tailwind -By exactly this code @media (min-width: 768px){} this is completely deleted after I insert my style.

**Comment by adamwathan** [comment]
Hey! For issues we need a reproduction, as mentioned in the template.
